netket/graph/abstract_graph.py

```diff
@@ -46,8 +46,8 @@ def distances(self) -> List[List]:
         raise NotImplementedError
 
     @abc.abstractmethod
-    def automorphisms(self):
-        r"""Symmetry group containing the automorphisms of the graph"""
+    def _automorphisms(self):
+        r"""Lists the automorphisms of the graph"""
```

Comment:
To encourage users to use the `automorphism_group()` method that went to live elsewhere (because circular imports, see in the long post). We can revert it to `automorphisms`

Reply:
It should be kept with a `@deprecated` decorator, at least. This was part of the released API for quite a while.

Reply:
at least you need to deprecate-warn it.